package.json

```diff
@@ -25,7 +24,7 @@
   },
   "homepage": "https://github.com/gcanti/fp-ts-rxjs",
   "dependencies": {
-    "fp-ts": "^1.0.1",
+    "fp-ts": "^1.5.0",
```

Comment:
Upgrading `typescript` is ok, not sure why you want to change the supported versions of `fp-ts` though

Reply:
in "dependencies" yes